src/service/navigation.js

```diff
@@ -124,6 +124,9 @@ export class Navigation {
      * @private {?Array<string>}
      */
     this.a2aFeatures_ = null;
+
+    /** @const @private {!Array<!AnchorMutator>} */
```

Comment:
`ClickListener`?

Reply:
From your comment on using sparse arrays and the code sample..I take it that you wanted !Array<function> and just do away with the AnchorMutator object.